src/libraries/System.Runtime.Extensions/tests/System/Convert.FromHexString.cs

```diff
@@ -34,93 +35,125 @@ public static void CompleteValueRange()
 
         private static void TestSequence(byte[] expected, string actual)
         {
-            Assert.Equal(expected, Convert.FromHexString(actual));
+            byte[] fromResult = Convert.FromHexString(actual);
+            Assert.Equal(expected, fromResult);
+
+            Span<byte> tryResult = stackalloc byte[actual.Length / 2];
```

Comment:
using `new` will be more future-proof (if you provide a large input, the test will crash with StackOverflowException)

```suggestion
            Span<byte> tryResult = new byte[actual.Length / 2];
```